### PR TITLE
Updating README to reflect capabilities of current version

### PR DESCRIPTION
**Pull request by a17levine**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,32 +23,15 @@ It also requires a valid `[API_KEY]`, which authenticates requests to the API. Y
...
-client = SwiftypeAppSearch::Client.new(:host_identifier => 'host-c5s2mj', :api_key => 'api-mu75psc5egt9ppzuycnc2mc3')
+client = SwiftypeAppSearch::Client.new(:account_host_key => 'host-c5s2mj', :api_key => 'private-mu75psc5egt9ppzuycnc2mc3')
 ```
 
 ### API Methods
 
 This client is a thin interface to the Swiftype App Search Api. Additional details for requests and responses can be
 found in the [documentation](https://swiftype.com/documentation/app-search).
 
-#### Indexing: Creating or Updating a Single Document
```
Comment (tehsven) [inline README.md]: Thanks for the submission @a17levine ! Indexing a single document exists, so this section should not be removed from the README - https://github.com/swiftype/swiftype-app-search-ruby/blob/master/lib/swiftype-app-search/client/documents.rb#L28

Comment (a17levine) [reply README.md]: @tehsven Hm interesting. It didn't show up when I did a `.methods` on the client object. I'll put it back

Comment (a17levine) [reply README.md]: Maybe its related to my environment

File: README.md [hunk 2]
```diff
@@ -23,32 +23,15 @@ It also requires a valid `[API_KEY]`, which authenticates requests to the API. Y
 You can find your `[API_KEY]` and your `[HOST_IDENTIFIER]` within the [Credentials](https://app.swiftype.com/as/credentials) menu:
 
 ```ruby
-client = SwiftypeAppSearch::Client.new(:host_identifier => 'host-c5s2mj', :api_key => 'api-mu75psc5egt9ppzuycnc2mc3')
+client = SwiftypeAppSearch::Client.new(:account_host_key => 'host-c5s2mj', :api_key => 'private-mu75psc5egt9ppzuycnc2mc3')
```
Comment (JasonStoltz) [inline README.md]: This was actually updated recently to prefer `host_identifier` over `account_host_key`, which is how we are now referring this in documentation: https://github.com/swiftype/swiftype-app-search-ruby/commit/7abfa8c4878e14a5ebf05b4e5e1b32b648e8f098. Both are still supported, however.

Comment (a17levine) [reply README.md]: @JasonStoltz I got an error that claimed I needed `account_host_key`

Comment (a17levine) [reply README.md]: @JasonStoltz I'll change it back if you insist it works, I just spent a lot of time earlier trying to get it working and found these settings got it to work

Comment (JasonStoltz) [reply README.md]: @a17levine What version of the gem do you have? If you have an older version than `0.1.4`, then that would be the case.

Comment (JasonStoltz) [reply README.md]: I'll double check on my end as well.

**Comment by JasonStoltz** [comment]
Closing this PR. We believe the documentation is correct as is. Thank you!

**Pull request closed without merging**